app/controllers/events_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class EventsController < ApplicationController
+  def index
+    @events = Event.all
+    @upcoming = Event.upcoming
+    @past = Event.past
+  end
+
+  def new
+    @event = current_user.events.build
+  end
+
+  def create
+    @event = Event.new(event_params)
```

Comment:
- [ ] You should be using the method [build](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/build_method_requirement.png) for the create action

Reply:
This one is fixed